pulp_smash/constants.py

```diff
@@ -248,6 +248,11 @@
 )
 """The absolute path to :data:`pulp_smash.constants.RPM` in the filesystem."""
 
+RPM_ENV_DESC = 'A package environment of Pulp tests.'
+
+RPM_ENV_NAME = 'Pulp Test Packages'
+"""The name and description of Package Environments."""
```

Comment:
Split this docstring and add the description part to the RPM_ENV_DESC and leave the name information here.